data/src/main/java/com/microsoft/azure/kusto/data/auth/HttpResponseWrapper.java

```diff
@@ -82,7 +83,11 @@ public Mono<byte[]> getBodyAsByteArray() {
 
         if (body == null) {
             try {
-                body = EntityUtils.toByteArray(response.getEntity());
+                if (response.getStatusLine().getStatusCode() == HttpStatus.SC_NO_CONTENT) {
+                    body = new byte[0];
```

Comment:
Minor nit, do you want to a constant EMPTY_BYTE_ARRAY

Reply:
agreed Ram, EMPTY_BYTE_ARRAY is a cleaner way of handling empty bye array.